src/tsx/timeline.tsx

```diff
@@ -195,7 +195,7 @@ class Timeline extends React.PureComponent<IProps, IState> {
           promptContainer
             .append("xhtml:div")
             .classed(style.promptText, true)
-            .text(({ prompt }) => prompt);
+            .html(({ prompt }) => boldBetweenBrackets(prompt));
```

Comment:
This is now vulnerable to HTML injection, but I think that's fine since we trust the opus writer.